module/behaviour/Controller/src/Controller.cpp

```diff
@@ -92,7 +92,9 @@ namespace module::behaviour {
             [this](const ActionPriorities& update) {
                 auto& request = requests[update.id];
 
-                // Find the largest priority
+                // TODO(BehaviourTeam): This can be converted to min_element
+                // ! in order to have the highest priority being
+                // ! 0 rather than the upper limit being extremely high Find the largest priority
```

Comment:
I would disagree, it's not really that big of a deal and we have the problem the other way around anyway. Get up/relax are at one end of the spectrum, stand is at the other end (priority epsilon or something).